usaspending_api/awards/v2/views/idvs/activity.py

```diff
@@ -78,11 +80,49 @@
     where   {award_id_column} = {award_id}
 """)
 
+COUNT_ACTIVITY_HIDDEN_SQL = SQL("""
+    with gather_award_ids as (
+        select  award_id,
+                false grandchild
+        from    parent_award
+        where   {award_id_column} = {award_id}
+        union all
+        select  cpa.award_id,
+                true grandchild
+        from    parent_award ppa
+                inner join parent_award cpa on cpa.parent_award_id = ppa.award_id
+        where   ppa.{award_id_column} = {award_id}
+    )
+    select
+        count(ca.id) rollup_contract_count
+    from
+        gather_award_ids gaids
+        inner join awards pa on pa.id = gaids.award_id
+        inner join awards ca on
+            ca.parent_award_piid = pa.piid and
+            ca.fpds_parent_agency_id = pa.fpds_agency_id and
+            ca.type not like 'IDV%'
+            {hide_edges_awarded_amount}
+        left outer join transaction_fpds tf on tf.transaction_id = ca.latest_transaction_id
+        left outer join recipient_lookup rl on rl.duns = tf.awardee_or_recipient_uniqu
+        left outer join agency a on a.id = ca.awarding_agency_id
+        left outer join toptier_agency ta on ta.toptier_agency_id = a.toptier_agency_id
```

Comment:
Since all we're really counting here are the child awards (`ca`) we don't need any of these `left outer joins` except for the `transaction_fpds` (since it can show up in the `WHERE` clause).